tests/components/asuswrt/test_sensor.py

```diff
@@ -122,7 +127,7 @@ async def test_sensors(hass, connect):
     # initial devices setup
     assert await hass.config_entries.async_setup(config_entry.entry_id)
     await hass.async_block_till_done()
-    async_fire_time_changed(hass, utcnow() + timedelta(seconds=30))
+    async_fire_time_changed(hass, now() + timedelta(seconds=30))
```

Comment:
The internal clock in Home Assistant is based on UTC time. Using UTC time here shouldn't be a problem.

Reply:
Let me say that this with utcnow() was not working, I discovered issue after changing the way `MOCK_DEVICES` was returned in test, because before the test was modifying directly the value in memory (don't know if I explained clearly the concept 😒)

Reply:
Please explain more in detail what the problem is.

Reply:
Did some test again, you are right `utcnow` work properly so I revert this change, Don't know why I had issue in the past, probably result of multiple tentative.
The only change here is the copy of `MOCK_DEVICE` before returning it. The reason for this is to avoid issue in case that the structure in memory is manipulated by the `router` class (e.g. performing a `pop` or a `clean` in the variable during elaboration).